doc/user/content/ingest-data/postgres-amazon-rds.md

```diff
@@ -0,0 +1,472 @@
+
+    ```sql
+    CREATE CLUSTER ingest_rds_postgres REPLICAS (r1 (SIZE = 'medium'));
+    ```
+
+    We recommend starting with a `medium` [size](/sql/create-cluster-replica/#sizes) replica or larger. This will help Materialize more quickly take an initial snapshot of the tables in your publication. Once the snapshot is finished, you'll right-size the cluster replica.
+
+## Step 5. Start ingesting data
+
+Now that you've configured your database network and created a source cluster, you can connect Materialize to your PostgreSQL database and start ingesting data. The exact steps depend on your networking configuration, so start by selecting the relevant option.
+
+{{< tabs >}}
+
+{{< tab "Allow Materialize IPs">}}
+
+1. [Earlier](#step-2-create-a-publication), you created a `materialize` user in your PostgreSQL database. Use the [`CREATE SECRET`](/sql/create-secret/) command to securely store its associated password in Materialize:
+
+    ```sql
+    CREATE SECRET pgpass AS '<PASSWORD>';
+    ```
+
+1. Use the [`CREATE CONNECTION`](/sql/create-connection/) command to create a connection object with access and authentication details for Materialize to use:
+
+    ```sql
+    CREATE CONNECTION pg_connection TO POSTGRES (
+      HOST '<host>',
+      PORT 5432,
+      USER 'materialize',
+      PASSWORD SECRET pgpass,
+      SSL MODE 'require',
+      DATABASE '<database>'
+      );
+    ```
+
+    Replace `<host>` with the your RDS endpoint. To find your RDS endpoint, select your database in the RDS Console, and look under **Connectivity & security**.
+
+    Replace `<database>` with the name of the database containing the tables you want to replicate to Materialize.
+
+1. Use the [`CREATE SOURCE`](/sql/create-source/) command to connect Materialize to your RDS instance and start ingesting data from the publication you created in PostgreSQL earlier:
+
+    ```sql
+    CREATE SOURCE mz_source
+      IN CLUSTER ingest_rds_postgres
+      FROM POSTGRES CONNECTION pg_connection (PUBLICATION 'mz_source')
+      FOR ALL TABLES;
+    ```
+
+    To ingest data from specific schemas or tables in your publication, use `FOR SCHEMAS (<schema1>,<schema2>)` or `FOR TABLES (<table1>, <table2>)` instead of `FOR ALL TABLES`.
+
+{{< /tab >}}
+
+{{< tab "Use AWS PrivateLink">}}
+
+1. Use the [`CREATE CONNECTION`](/sql/create-connection/#aws-privatelink) command to create an AWS PrivateLink connection:
+
+    ```sql
+    CREATE CONNECTION privatelink_svc TO AWS PRIVATELINK (
+      SERVICE NAME 'com.amazonaws.vpce.us-east-1.vpce-svc-0356210a8a432d9e9',
+      AVAILABILITY ZONES ('use1-az1', 'use1-az2', 'use1-az3')
+    );
+    ```
+
+    Replace the `SERVICE NAME` value with the service name you noted [earlier](#step-3-configure-network-security).
+
+    Replace the `AVAILABILITY ZONES` list with the IDs of the availability zones in your AWS account.
```

Comment:
It's not straight forward to find these and a lot of folks will initially think that this is asking for the availability zone rather than the availability zone id. I'm not sure what is the best way to find it, but one way to find it is to navigate to the subnets of the VPC and copy the `Availability Zone ID` from there. I think we should spell this out here.

Reply:
Done.